setup.cfg

```diff
@@ -1,4 +1,62 @@
+[metadata]
+name = tiledb-ml
+version = 0.1.0
+description = Package supports all machine learning functionality for TileDB Embedded and TileDB Cloud
+author = TileDB, Inc.
+author_email = help@tiledb.io
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = MIT
+license_file = LICENSE
+keywords = tiledb, ml
+url = https://github.com/TileDB-Inc/TileDB-ML
+platform = any
+project_urls =
+    Bug Tracker = https://github.com/TileDB-Inc/TileDB-ML/issues
```

Comment:
Not sure if the space in `Bug Tracker` is parsed correctly

Reply:
It should be ok imo. If you check in [TestPypi](https://test.pypi.org/project/tiledb-ml/) the package `0.2.0` is the result of this PR.